nixos/modules/services/hardware/amd-hybrid-graphics.nix

```diff
@@ -25,15 +25,22 @@
       path = [ pkgs.bash ];
       description = "Disable AMD Card";
       after = [ "sys-kernel-debug.mount" ];
-      requires = [ "sys-kernel-debug.mount" ];
-      wantedBy = [ "multi-user.target" ];
+      before = [ "systemd-vconsole-setup.service" "display-manager.service" ];
```

Comment:
Hm, I'm a litt confused about the dependency graph here; previously this unit would be pulled in by the default target, but what pulls it in now?